concent_api/core/tests/test_unit_message_handlers.py

```diff
@@ -2,8 +2,13 @@
 
 from django.conf import settings
 from django.test import override_settings
+
+from common import constants
+from core.exceptions import Http400
```

Comment:
@dybi would point to change it to some custom exception :P There are some validation exceptions that inherit from `ConcentBaseException`. Use one of those or create new that also inherit from it.

Reply:
Yeah, we were supposed to start using more specific exceptions.

Reply:
I also think so, the reason I used http400 was, we used it in almost all places and similar situations. For this `GolemMessageValidationError` will be perfect